static/app/components/sidebar/sidebarDropdown/switchOrganization.spec.tsx

```diff
@@ -92,11 +92,11 @@ describe('SwitchOrganization', function () {
 
     expect(screen.getByRole('list')).toBeInTheDocument();
 
-    const org1Link = screen.getByRole('link', {name: 'org1 Organization 1'});
+    const org1Link = screen.getByRole('link', {name: 'O Organization 1'});
```

Comment:
i think it was able to get the aria label on the svg before.
now it is the literal text of the avatar